src/francium/infos.ts

```diff
@@ -245,7 +245,7 @@ infos = class InstanceFrancium {
 
   static async getFarmerId(farmInfo: types.FarmInfo, userKey: PublicKey, version?: number): Promise<PublicKey> {
     const ata = await getAssociatedTokenAddress(farmInfo.stakedTokenMint, userKey);
-    const [farmInfoPub, nonce] = await PublicKey.findProgramAddress(
+    const [farmInfoPub, nonce] = await PublicKey.findProgramAddressSync(
```

Comment:
Hey @wei1769 I noticed that you left in the await here, I think it might be redundant.

Reply:
There's a few more instances of similar uses of ```await``` with Synchronous function calls

Reply:
Ayush and I were on call and going through the code, and he pointed out this in the documentation: https://solana-labs.github.io/solana-web3.js/classes/PublicKey.html#findProgramAddressSync